starter-code/index.html

```diff
@@ -0,0 +1,253 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+        integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <link rel="stylesheet" href="../starter-code/style.css">
+    <title>Lab Bootstrap Cloning Revera</title>
+</head>
+
+<body>
+    <heading>
+        <div class="row">
+            <div id="home page" class="container-fluid">
+                <div id="general top menu">
+                    <nav class="navbar navbar-expand-lg navbar-light bg-white">
```

Comment:
There is a lot padding going on here, due to the class bg-white.
Could the padding be decreased?
Please mind that bg-white is used in other places, so altering its padding values, will also have consequences there.